### DISCUSSION TITLE
Am I adding an application correctly?

### DISCUSSION THREAD

**Discussion by jdev082** [ScoopInstaller/Main#4053] [Q&A]
So I made a PR here: https://github.com/ScoopInstaller/Main/pull/4052

I also see issues with package requests, was I supposed to post there first?

**Comment by niheaven** [comment]
See here: https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket

Personally I think your app should be added in `extras` bucket and using nupkg as installation package.

**Reply by jdev082** [reply]
@niheaven it will have to stay on my personal bucket because the 'extras' still requires that it is [b]not[/b] a GUI application.

**Reply by niheaven** [reply]
Aha? non-GUI? That's for `main`, not `extras`.
